docs/sources/getting-started/_index.md

```diff
@@ -1,8 +1,88 @@
 +++
-title = "Getting started"
+title = "Getting Started"
```

Comment:
-1, please use sentence-style headings.

Reply:
since "/docs/grafana/latest/guides/what-is-grafana" page is being removed would be good to add an alias here for that page so users / old links / search get's redirected to this page instead of a 404 not found page